content-src/lib/ReduxChannel.js

```diff
@@ -42,16 +42,17 @@ class Channel {
   }
 
   connectStore(store) {
-    this.on(action => store.dispatch(action));
+    this.on((action) => store.dispatch(action));
   }
 
   get middleware() {
-    return store => next => function(action) {
+    return (store) => (next) => function(action) {
       const meta = action.meta || {};
       const timeouts = this.timeouts;
 
       // Send action to the next step in the middleware
-      next(action);
+      // TODO: "Expected return with your callback function"
+      next(action); // eslint-disable-line callback-return
```

Comment:
This was an interesting one. ESLint seems to think that since we're calling `next()` we should have a return statement here (which sounds solid if we're talking about middleware, callbacks, or promises). I'm not familiar with the code to know if this is a bug or not, so I just disabled the rule here and added a "TODO".